### PR TITLE
[jboss-logging] initial integration

### PR DESCRIPTION
**Pull request by wojtekCodeInt**

### CODE REVIEW

File: infra/helper.py [hunk 1]
```diff
@@ -669,7 +669,7 @@ def docker_run(run_args, print_output=True, architecture='x86_64'):
   """Calls `docker run`."""
   platform = 'linux/arm64' if architecture == 'aarch64' else 'linux/amd64'
   command = [
-      'docker', 'run', '--rm', '--privileged', '--shm-size=2g', '--platform',
+      'docker', 'run', '--privileged', '--shm-size=2g', '--platform',
```
Comment (jonathanmetzman) [inline infra/helper.py:672]: This change is a mistake right?

Comment (henryrneh) [reply infra/helper.py]: Done

File: projects/jboss-logging/Dockerfile [hunk 2]
```diff
@@ -0,0 +1,33 @@
+# Copyright 2021 Google LLC
```
[the remark this answers is not in the document]
Comment (henryrneh) [reply projects/jboss-logging/Dockerfile]: Done

File: projects/jboss-logging/build.sh [hunk 3]
```diff
@@ -0,0 +1,80 @@
+#!/bin/bash -eu
+# Copyright 2021 Google Inc.
```
[the remark this answers is not in the document]
Comment (henryrneh) [reply projects/jboss-logging/build.sh]: Done

File: projects/jboss-logging/project-parent/fuzz-targets/src/test/java/com/example/FFuzzer.java [hunk 4]
```diff
@@ -0,0 +1,43 @@
+// Copyright 2022 Google LLC
```
[the remark this answers is not in the document]
Comment (henryrneh) [reply projects/jboss-logging/project-parent/fuzz-targets/src/test/java/com/example/FFuzzer.java]: Done

**Comment by jonathanmetzman** [comment]
This ready to be merged?

File: projects/jboss-logging/build.sh [hunk 5]
```diff
@@ -0,0 +1,80 @@
+#!/bin/bash -eu
+# Copyright 2023 Google LLC.
```
Comment (jonathanmetzman) [inline projects/jboss-logging/build.sh:2]: nit: "LLC" not "LLC."

Comment (henryrneh) [reply projects/jboss-logging/build.sh]: Thanks for the correction!

**Comment by henryrneh** [comment]
> This ready to be merged?

@wojtekCodeInt can you press the button and make this PR ready for review?

**Comment by 0roman** [comment]
@oliverchang

**Comment by Alan32Liu** [comment]
Enabling auto-merge as I presume this is ready to be merged?

**Review by Alan32Liu — APPROVED** [review]

**Pull request merged**